newsletter/index.html

```diff
@@ -0,0 +1,69 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+
+    <!-- Standard Metadata -->
+    <title>Embark Studios Open Source Newsletter | embark.dev</title>
+    <meta name="description" content="Discover open source repositories in Rust, Go, and more from Embark Studios">
```

Comment:
We can change the description (here and in line 16) to something more relevant to the page.
something along the lines of "stay up to date with blog posts, open source, and events with the developer newsletter from Embark Studios."